client/client.go

```diff
@@ -1,6 +1,7 @@
 package client
 
 import (
+	"github.com/openshift/client-go/apps/clientset/versioned"
```

Comment:
Minor stylistic issue, it might be nicer to alias this as something more explicit, e.g. openshiftapps or something.